msteams-platform/m365-apps/overview-combining-add-in-and-teams-app.md

```diff
@@ -0,0 +1,85 @@
+- Use the Teams app project as the base project for the merged application.
+- Keep configuration files that are applicable to both the add-in and Teams app in the root of the project.
+- Move source files and configuration files that are only applicable to the Teams app into a subfolder off the root of the project. Similarly, copy source and configuration files that are only used by the add-in from the existing add-in to a different subfolder off the root of the combined project.
+- Use the same base domain for the built files of both the add-in and Teams app, for example contoso.com.
+- Merge the two manifests into a single manifest. Use the process you went through in step 1 as a guide.
+- The [id](../resources/schema/manifest-schema.md#id) property in the new manifest should be the same value as the "id" property in the original Teams app manifest.
+- Raise the value of the [version](../resources/schema/manifest-schema.md#version) property; for example, raise "1.0.0" to "1.1.0".
+- Add an [extensions.alternates](../resources/schema/manifest-schema.md#extensionsalternates) property to the manifest and configure it to hide the original Office add-in in versions of Office that support combining an add-in and a Teams app. For more information, see [Manage new and old versions of an add-in](/office/dev/add-ins/concepts/duplicate-legacy-metaos-add-ins).
+
+## Publish the combined Teams app and Office add-in
+
+To publish the combined app, treat it like an update to the Teams app. For more information, see [Publish updates to your app](../concepts/deploy-and-publish/appsource/post-publish/overview.md#publish-updates-to-your-app).
+
+When the update is accepted, how it becomes available to end users varies. The following are the general principles.
+
+- Users who had previously acquired both the Teams app and the Office add-in have the new version of both available immediately, if the Microsoft 365 administrator consents to the update. Otherwise, users are prompted to consent.
+- Users who had previously acquired only the Teams app have the new version of *both* available immediately, if the Microsoft 365 administrator consents to the update. Otherwise, users are prompted to consent.
+- Users who had previously acquired only the Office add-in must acquire the new combined app as a Teams app.
+
+> [!IMPORTANT]
+> Users with certain older versions of Office may end up with the old version of the add-in, even after the new combined app is acquired. Generally, add-ins that use the unified app manifest for Microsoft 365 (formerly Teams app manifest) can be installed only on Microsoft 365 Version 2307 (Build 16626.20132) and later. However, there are two exceptions which enable these add-ins to be installed on older versions of Microsoft 365 and on perpetual license versions of Office.
```

Comment:
Would want to put it somewhat like this - 
Users on certain older client may not be able to acquire the new app. However, if the new app is acquired via OWA or Win32 version 2307 or later OR the unified app is deployed by an admin for them, the add-in component of the new app will be available to them. This means that as a developer you can publish updates to only the new version of the app and it can be made available to the users on older clients as well which do not support unified apps.

Reply:
I will add the bit about OWA and updating only the new version, but otherwise I prefer my wording.